docs/sdks/server-sdks/dotnet.md

```diff
@@ -49,60 +75,262 @@ internal class AssignmentLogger : IAssignmentLogger
+| `BanditKey` (string)                                 | The key (unique identifier) of the bandit                                                                         | "ad-bandit-1"                    |
+| `SubjectKey` (string)                                | An identifier of the subject or user assigned to the experiment variation                                         | "ed6f85019080"                   |
+| `SubjectNumericAttributes` (IDict<String,double>)    | Metadata about numeric attributes of the subject. Map of the name of attributes their provided values             | `{"age": 30}`                    |
+| `SubjectCategoricalAttributes` (IDict<String,string) | Metadata about non-numeric attributes of the subject. Map of the name of attributes their provided values         | `{"loyalty_tier": "gold"}`       |
+| `Action` (string)                                    | The action assigned by the bandit                                                                                 | "promo-20%-off"                  |
+| `ActionNumericAttributes` (IDict<String,double)      | Metadata about numeric attributes of the assigned action. Map of the name of attributes their provided values     | `{"brandAffinity": 0.2}`         |
+| `ActionCategoricalAttributes` (IDict<String,string)  | Metadata about non-numeric attributes of the assigned action. Map of the name of attributes their provided values | `{"previouslyPurchased": false}` |
+| `ActionProbability` (number)                         | The weight between 0 and 1 the bandit valued the assigned action                                                  | 0.25                             |
+| `OptimalityGap` (number)                             | The difference between the score of the selected action and the highest-scored action                             | 456                              |
+| `ModelVersion` (string)                              | Unique identifier for the version (iteration) of the bandit parameters used to determine the action probability   | "v123"                           |
+| `MetaData` IDict<string, string>                     | Any additional freeform meta data, such as the version of the SDK                                                 | `{ "sdkLibVersion": "3.5.1" }`   |
+
+
+### Querying for a Bandit Action
+To query the bandit for an action, there are multiple overloads of the `getBanditAction()` function you can use.  This function takes the following parameters:
+- `flagKey` (string): The key of the feature flag corresponding to the bandit
+- `subject` (ContextAttributes): The key of the subject or user assigned to the experiment variation along with the subject's Categorical and Numeric attributes
+- `actions` (IDictionary<string, ContextAttributes>): Map of actions (by name) to their Context (Categorical and Numeric)
+- `defaultValue` (string): The default *variation* to return if the flag is not successfully evaluated
```

Comment:
or--the more common scenario--the flag is disabled.

`The default *variation* to return if the flag is disabled or not successfully evaluated`

Reply:
done